src/vs/editor/common/diff/defaultLinesDiffComputer/heuristicSequenceOptimizations.ts

```diff
@@ -113,6 +122,33 @@ function joinSequenceDiffsByShifting(sequence1: ISequence, sequence2: ISequence,
 	return result2;
 }
 
+/**
+ * fixes issues like this:
+ * seq1: [    pass] [    pass]
+ * seq2: [    pass] [  pass]
+ */
+function checkSequencesIsStronglyEqual(sequence1: ISequence, seq1OffsetS: number, seq1OffsetE: number, sequence2: ISequence, seq2OffsetS: number, seq2OffsetE: number): boolean {
```

Comment:
I'm not sure I undestand what this function is supposed to do.

Reply:
You're right, this method doesn't require.